Project.toml

```diff
@@ -17,22 +17,22 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
-Adapt = "3.3.3, 4"
-Aqua = "0.8"
-CellularAutomata = "0.0.2"
-DifferentialEquations = "7"
-Distances = "0.10"
-Distributions = "0.24, 0.25"
-LIBSVM = "0.8"
-LinearAlgebra = "1"
-MLJLinearModels = "0.5, 0.6, 0.7, 0.8, 0.9"
-NNlib = "0.8.4, 0.9"
-Optim = "1"
-Random = "1"
-SafeTestsets = "0.1"
-SparseArrays = "1"
-Statistics = "1"
-Test = "1"
+Adapt = "~3.3.3"
+Aqua = "=0.8.0"
+CellularAutomata = "=0.0.2"
+DifferentialEquations = "~7.0.0"
+Distances = "=0.10.0"
+Distributions = "=0.24.5"
+LIBSVM = "=0.8.0"
+LinearAlgebra = "~1.9.0"
+MLJLinearModels = "=0.9.2"
```

Comment:
All of these `=` seem to be extra strict?

Reply:
I accidentally staged the end result of running the downgrade script locally.